Source/Core/DolphinQt/Settings/GeneralPane.cpp

```diff
@@ -471,3 +459,39 @@ void GeneralPane::AddDescriptions()
   m_button_generate_new_identity->SetDescription(tr(TR_GENERATE_NEW_IDENTITY_DESCRIPTION));
 #endif
 }
+
+void GeneralPane::UpdateDescriptionsUsingHardcoreStatus(const bool enabled)
+{
+  static constexpr char TR_CHEATS_DESCRIPTION[] = QT_TR_NOOP(
+      "Enables the use of AR and Gecko cheat codes which can be used to modify games' behavior. "
+      "These codes can be configured with the Cheats Manager in the Tools menu."
+      "<br><br>This setting cannot be changed while emulation is active."
+      "<br><br><dolphin_emphasis>If unsure, leave this unchecked.</dolphin_emphasis>");
+  static constexpr char TR_SPEEDLIMIT_DESCRIPTION[] =
+      QT_TR_NOOP("Controls how fast emulation runs relative to the original hardware."
+                 "<br><br>Values higher than 100% will emulate faster than the original hardware "
+                 "can run, if your hardware is able to keep up. Values lower than 100% will slow "
+                 "emulation instead. Unlimited will emulate as fast as your hardware is able to."
+                 "<br><br><dolphin_emphasis>If unsure, select 100%.</dolphin_emphasis>");
+  static constexpr char TR_DISABLED_IN_HARDCORE_DESCRIPTION[] =
+      QT_TR_NOOP("<dolphin_emphasis>Disabled in Hardcore Mode.</dolphin_emphasis>");
+  static constexpr char TR_SPEEDLIMIT_RESTRICTION_IN_HARDCORE_DESCRIPTION[] =
+      QT_TR_NOOP("<dolphin_emphasis>When Hardcore Mode is enabled, Speed Limit values less than "
+                 "100% will not slow emulation.</dolphin_emphasis>");
```

Comment:
Maybe this would be clearer:

```suggestion
      QT_TR_NOOP("<dolphin_emphasis>When Hardcore Mode is enabled, Speed Limit values less than "
                 "100% will be treated as 100%.</dolphin_emphasis>");
```